### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by Hosch250**

### CODE REVIEW

File: RetailCoder.VBE/UI/Command/AddTestModuleCommand.cs [hunk 1]
```diff
@@ -16,7 +15,7 @@ public class AddTestModuleCommand : CommandBase
         private readonly VBE _vbe;
         private readonly NewUnitTestModuleCommand _command;
 
-        public AddTestModuleCommand(VBE vbe, RubberduckParserState state, NewUnitTestModuleCommand command) : base(LogManager.GetCurrentClassLogger())
+        public AddTestModuleCommand(VBE vbe, NewUnitTestModuleCommand command) : base(LogManager.GetCurrentClassLogger())
```
Comment (retailcoder) [inline RetailCoder.VBE/UI/Command/AddTestModuleCommand.cs]: What's the difference between `AddTestModuleCommand`, and `NewUnitTestModuleCommand`? Shouldn't one *be* the other, and the other killed off?

Comment (Hosch250) [inline RetailCoder.VBE/UI/Command/AddTestModuleCommand.cs]: The second is the one that actually adds either test to the module.  The first (and it's expected error equivalent) both just call the other.

Comment (retailcoder) [inline RetailCoder.VBE/UI/Command/AddTestModuleCommand.cs]: That's my point, one of the two seems redundant...

Comment (Hosch250) [inline RetailCoder.VBE/UI/Command/AddTestModuleCommand.cs]: Yeah, same here.  Let's see what I can do.

**Pull request merged**